src/tests/ftest/config_file_gen.py

```diff
@@ -51,6 +51,12 @@ def generate_server_config(args):
     config.engine_params[0].storage.storage_tiers[0].storage_class.value = "ram"
     config.engine_params[0].storage.storage_tiers[0].scm_mount.value = "/mnt/daos"
     config.engine_params[0].storage.storage_tiers[0].scm_size.value = 0
+
+    # Update PMem related env variables to disable NDCTL for non-dcpm storage.
+    # PMEMOBJ_CONF=sds.at_create=0 disable NDCTL-related features of PMDK.
+    # NDCTL features cause test failures if PMDK is used with RAM-based simulated PMem
```

Comment:
I am trying to make this comment future-proof. My thinking is:

- It is not `PMem related env`. It is PMEMOBJ-specific.
- It is hard to follow why NDCTL is even mentioned here.
- It is not explained what `NDCTL-related features` are.

My suggestion would be:

```python
    # Disable shutdown state (SDS) for RAM-based simulated SCM.
    # RAM doesn't support this feature and trying to use it will fail the create/open operations.
```

Open to discussion.